README.md

```diff
@@ -1,64 +1,77 @@
+| Documentation      | [![Documentation Status](https://readthedocs.org/projects/eit_dash/badge/?version=latest)](https://eit_dash.readthedocs.io/en/latest/?badge=latest)                                                                                            |
 
-## Mockup
-
-[mockup](https://github.com/EIT-ALIVE/eitprocessing/files/11480259/Proposal.GUI.mockup.pptx)
-
-## How to use eit_dash
-
-GUI to load and analyze image data from electrical impedance tomography (EIT)
+## Getting started
 
-The project setup is documented in [project_setup.md](project_setup.md). Feel free to remove this document (and/or the link to this document) if you don't need it.
+### 1. Installation
 
-## Getting started
+##### Install Poetry
 
-### Installing Poetry
+EIT Dashboard uses of [poetry](https://python-poetry.org/) to easily manage the needed packages.
+Poetry can be installed as follows. Please refer to the [official installation instructions](https://python-poetry.org/docs/#installation) if problems arise:
 
-EIT Dashboard makes use of [poetry](https://python-poetry.org/) to easily manage the needed packages. 
-`Poetry` can be installed both at system level following the [installation instructions](https://python-poetry.org/docs/#installation):
+In Linux (and WSL) or macOS
 
-In Linux, macOS, Windows (WSL)
 ```console
 curl -sSL https://install.python-poetry.org | python3 -
 ```
 
-In Windows (PowerShell)
+In Windows (using PowerShell)
+
 ```console
 (Invoke-WebRequest -Uri https://install.python-poetry.org -UseBasicParsing).Content | py -
 ```
 
-Alternatively, Poetry can also be installed [using pip](https://pypi.org/project/poetry/). 
+Alternatively, poetry can also be installed [using pip](https://pypi.org/project/poetry/) in a virtual environment of choice.
 
-### Creating the virtual environment 
+##### Install EIT Dashboard
 
-The first time that the dashboard is used, the repository needs to be cloned and the needed dependencies have to be installed by running:
+The first time that the dashboard is used, the repository needs to be cloned and the needed dependencies have to be
+installed by navigating to the path where it should be installed and running:
 
 ```console
 git clone git@github.com:EIT-ALIVE/eit_dash.git
+cd eit_dash
+poetry install
+```
+
+### 2. Running EIT Dashboard
+
+##### Stay up to date
+
+To ensure you are using the newest version, including any updates since you last used it, navigate to the folder where
+the dashboard is installed and run:
+
+```console
+git pull
 poetry install
 ```
 
-### Running the dashboard
+Please see our [user manual](docs/user_manual.md) for instructions on how to use the dashboard.
+
+##### Run dashboard
 
-Once the environment has been created, the dashboard can be run through:
+Run the command below and and click (or ctrl/cmd+click) on the link shown.
 
 ```console
 poetry run python eit_dash/main.py
 ```
 
```

Comment:
I missed the comment about ctrl/cmd+clicking the link. Maybe this should be a separate sentence, or you could add a screenshot/example console output to show it?

Also, ctrl/cmd+clicking does not work in all terminals. E.g. in the default Terminal app on macOS it does not work. It does work in iTerm2 (the terminal most terminal users use).

Reply:
is it better now?